deployment.rb

```diff
@@ -93,6 +93,10 @@ def load_configuration()
     'dynamo_properties_table'     => "#{rack_env}_properties",
     'dynamo_table_metadata_table'         => "#{rack_env}_table_metadata",
     'throttle_data_apis'          => [:staging, :adhoc, :test, :production].include?(rack_env),
+    'firebase_max_channel_writes_per_15_sec' => 300,
+    'firebase_max_channel_writes_per_60_sec' => 600,
+    'firebase_max_table_rows'     => 1000,
```

Comment:
Those get used in `rake firebase:set_constants`: https://github.com/code-dot-org/code-dot-org/blob/staging/lib/rake/firebase.rake#L18 I included this rake target as a late change in the previous PR to give a better picture of all the rake tasks we'd want related to firebase.